AB_Hands_On/.ipynb_checkpoints/AB_Hands_On-checkpoint.ipynb

```diff
@@ -0,0 +1,6615 @@
+{
```

Comment:
<p>Is this updated? Still says 20 where it should be 7.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>